File: js/data-structures/stack.js
class Stack {
    constructor() {
        this.collection = [];
    }
    add(item) {
        this.collection.push(item);
    }
    remove() {
        this.collection.pop();
    }
}

const coll = new Stack();
coll.add(1);
coll.add(2);
coll.add(3);
coll.add(4);
coll.add(5);
coll.remove();

console.log(coll);

File: csharp/data-structures/Stack.cs
using System;
using System.Collections;

class MyStack
{
    private Stack stack;
    public MyStack()
    {
        stack = new Stack();
    }
    public void Add(string val)
    {
        stack.Push(val);
    }
    public void Remove()
    {
        stack.Pop();
    }
    public void Clear()
    {
        stack.Clear();
    }
    public void PrintAll()
    {
        foreach (var elem in stack)
        {
            Console.WriteLine(elem);
        }
    }
}